pelican/log.py

```diff
@@ -92,10 +92,15 @@ class LimitFilter(logging.Filter):
     """
 
     _ignore = set()
+    _raised_messages = set()
     _threshold = 5
     _group_count = defaultdict(int)
 
     def filter(self, record):
+        # don't filter log messages when "debug" is enabled
+        logger_level = logging.getLogger().getEffectiveLevel()
+        if logger_level == logging.DEBUG:
+            return True
```

Comment:
In debug mode now all messages get through even the ones which previously got filter because of a threshold.
(Or did I miss something?)

---

Also I don't understand the intention behind this change.
I see myself only using this feature **in** debug mode to reduce the logging flood.